components/collapse/collapse-panel.component.ts

```diff
@@ -68,7 +78,7 @@ const NZ_CONFIG_MODULE_NAME: NzConfigKey = 'collapsePanel';
     class: 'ant-collapse-item',
     '[class.ant-collapse-no-arrow]': '!nzShowArrow',
     '[class.ant-collapse-item-active]': 'nzActive',
-    '[class.ant-collapse-item-disabled]': 'nzDisabled'
+    '[class.ant-collapse-item-disabled]': 'nzDisabled || nzCollapsible === "disabled"'
```

Comment:
What's the difference between `nzDisabled` and `nzCollapsible`, I think we don't have to support this attribute, or we can just reuse its css class. @hsuanxyz Do you have any suggestions?

Reply:
`ant-design` deprecated `disabled` in 4.x and used `collapsible` as a replacement for `disabled`, do we need to mark `disabled` as deprecated and use `collapsible` instead of `disabled` in 14.x